### ISSUE TITLE
Windows 11 Preview build 26300.8772 and later: NVDA announces "tab" when searching in Start menu with refined search experience enabled

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#20530]
### Brief summary

In Windows 11 Insider Preview build 26300.8772 and later, NVDA does not announce search results when searching in the refined search experience

### Steps to reproduce

1. Install Windows 11 Insider Preview Experimental (26H2) build 26300.8772 and later (you must be a Windows Insider).
2. With the build installed and the system restarted, open Start, type search keywords.

### Actual behavior

NVDA says "all tab selected". This is with refined search experience enabled (check Settings (Windows+I)/Windows Update/Windows Insider Program/Feature flags).

### Expected behavior

NVDA announces top search result.

### NVDA logs, crash dumps and other attachments

None

### NVDA type

installed copy

### NVDA version

Alpha-57029

### Have you tried any other versions of NVDA? If so, please report their behaviors.

Same behavoir (2026.1.1, 2026.2 beta)

### Windows version

Windows 11 Insider Preview Experimental/26H2 (build 26300.8935, AMD64)

### Name and version of other software in use when reproducing the issue

None

### Other information about your system

Used as a Windows Insider development workstation

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have run the System Accessibility Repair Tool and the issue still occurs

**Comment by josephsl** [comment]
Additional info:

Technical: state change events are fired by search tabs/categories which is handled by NVDA
Reproducible with Narrator: yes
Responsible party: Microsoft
Feedback submitted to the app vendor (Microsoft): yes (Feedback Hub link: https://aka.ms/AA11w2ul)
Feedback status: Microsoft is investigating (source: Feedback Hub status)

Thanks.

**Comment by josephsl** [comment]
Hi,

Update: the refined search experience is now available to release preview Insiders as of build 26200.9267. This means retail (stable) channel users will receive this change AND will experience the problem reported here as early as end of August with the optional update.

Thanks.
